model_prices_and_context_window.json

```diff
@@ -3396,6 +3396,15 @@
         "mode": "chat",
         "supports_function_calling": true
       },
+      "anyscale/mistralai/Mixtral-8x22B-Instruct-v0.1": {
+        "max_tokens": 65536,
+        "max_input_tokens": 65536,
+        "max_output_tokens": 65536,
+        "input_cost_per_token": 0.00000090, 
+        "output_cost_per_token": 0.00000090,
+        "litellm_provider": "anyscale", 
+        "mode": "chat"
```

Comment:
please add `source` , `supports_function_calling` for all of these

Reply:
Just set it up for the models which support it, either via tools or json schema.